src/panels/config/info/system-health-card.ts

```diff
@@ -128,14 +128,24 @@ class SystemHealthCard extends LitElement {
   }
 
   private _copyInfo(): void {
+    // We have to copy title text and content separately, because
+    // copying the whole <ha-card> would also copy the tooltip text.
     const selection = window.getSelection()!;
     selection.removeAllRanges();
 
-    const copyElement = this.shadowRoot?.querySelector(
-      "ha-card"
+    let copyElement = this.shadowRoot?.querySelector(
+      ".card-header-text"
```

Comment:
Since the header is translated, maybe better to not copy that and hard code it in English? Assuming this is for requesting support in the community?

Reply:
I would vote against that. Although most commonly it will be used for support, it is still part of the regular user interface and therefore I think should be properly translated. If there is a way to change it in the clipboard but leave the UI, then that might be an option.

Reply:
Yeah, I just meant in the clipboard data, not in the UI.

We could add a text element before copying and remove it after.